jdaviz/configs/default/plugins/plot_options/plot_options.py

```diff
@@ -507,6 +507,13 @@ def _update_stretch_histogram(self, msg={}):
             bqplot_clear_figure(self.stretch_histogram)
             return
 
+        spectrum_viewer_reference = getattr(
+            self.app._jdaviz_helper, '_default_spectrum_viewer_reference_name', None
+        )
+        if self.viewer.selected == spectrum_viewer_reference:
+            # don't update a histogram if the spectrum viewer is selected:
+            return

```

Comment:
should we just check the type of the viewer to ensure its an image viewer before continuing?

Reply:
Isn't that effectively the same?

Reply:
it would be more future proof for other configs and the possibility of being able to dynamically add or rename spectral viewers 🤷‍♂️

Reply:
Done!